### PR TITLE
Update license headers

### PR DESCRIPTION
**Pull request by erlswtshrt**
Fixes: #21 #22 #23 #24

@ivanbogdanov @jandolina

### CODE REVIEW

**Comment by kaelig** [comment]
You'll also want to rename LICENSE to LICENSE.txt and copy contents from https://github.com/salesforce-ux/licenses/blob/master/LICENSE.txt

**Comment by kaelig** [comment]
@erlswtshrt Note that the generated files that are shipped in the pod should also have license headers.

**Review by ivanbogdanov — COMMENTED** [review]
@erlswtshrt can you resolve merge conflict before merging?

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2017-present, Salesforce.com, Inc.
```
Comment (kaelig) [inline LICENSE.txt]: This should probably say 2015-present

**Pull request merged**